en/advanced_config/ethernet_setup.md

```diff
@@ -0,0 +1,85 @@
+# PX4 Ethernet Setup
+
+Ethernet connectivity a faster, more reliable, and more flexible communication technology than serial communication approach.
+This method is preferred over serial communication in case of direct connection between Pixhawk and IP Radio telemetry, companion computers and Networks.
+It utilizes UDP connection for MAVLINK communication over the port 14550 with [MAVLINK2](https://mavlink.io/en/guide/mavlink_2.html). 
+
+## Supported Flight Controllers
```

Comment:
FYI only, here I've used headings to give the document structure and highlight relevant information.

Reply:
PS Note, also I refer to flight controllers generally here, not just Pixhawk or FMU 5X. It is likely that in future there will non FMU devices that still integrate with Ethernet.

Reply:
Thanks @hamishwillee . 
I changed to FCU.